code/modules/cm_marines/dropship_equipment.dm

```diff
@@ -922,7 +921,7 @@
 		return
 	if(!ship_base) //not installed
 		return
-	if(!skillcheck(user, SKILL_PILOT, SKILL_PILOT_TRAINED))
+	if(!skillcheck(user, SKILL_PILOT, SKILL_PILOT_TRAINED) && !skillcheck(user, SKILL_MEDICAL, SKILL_MEDICAL_DOCTOR))
```

Comment:
It should be OR not AND

Reply:
No, it is "if NEITHER". If both not true, then send "you can't do that" and return.

Reply:
have you tested that ? i think its should be II

Reply:
but if it works like that just mark this as resolved . maybe i am just reading wrong

Reply:
Again, it returns TRUE if user doesn't have neither pilot skills NOR doctor skills. So if they have pilot skills or doctor skills it returns FALSE. If I change it to || it will return TRUE if user doesn't have pilot skills OR doctor skills. This, doctor won't be able to operate it as they don't have pilot skills and vice versa. And my goal is opposite.